packages/testing-utils/types/index.d.ts

```diff
@@ -107,7 +107,6 @@ export interface SagaIteratorClone extends SagaIterator {
 export function createMockTask(): MockTask
 
 export interface MockTask extends Task {
-  setRunning(running: boolean): void
```

Comment:
I totally agree with this change - I am concerned about this being a breaking change though. I'm not opposed to releasing a new major version of the Redux Saga or accompanying packages but this seems like a minor cleanup that doesn't necessarily require us to immediately bump a major version.

This PR will probably change the behavior of this task mock slightly, one way or another. But removing a known method is a big breaking change (for this package alone). Could we think about something less disruptive here? My preference would be to add this method back and introduce `@deprecated` + `console.warn` for what we want to phase out.